maps/vgstation.dmm

```diff
@@ -0,0 +1,14 @@
+"a" = (/turf/space,/area)
```

Comment:
Waaaaaaaaait we use tgstation... I did my changes and pushed the wrong map file :^)